.github/workflows/python_uniffi_ci_cd.yml

```diff
@@ -54,17 +55,24 @@ jobs:
         if: inputs.release
         with:
           bun-version: latest
-      - run: cd ../../../ && bun install
+      - working-directory: '.'
+        run: bun install
         if: inputs.release
       - name: Semantic Release (dry-run)
         if: inputs.release
         id: semantic_release
         env:
           GITHUB_TOKEN: ${{ steps.app_token.outputs.token }}
-        run: "bun semantic-release --dry-run"
+        run: bun semantic-release --dry-run
+      - name: Convert to Python Version
+        if: steps.semantic_release.outputs.version != ''
+        id: convert_version
+        uses: ./.github/actions/as-python-version
+        with:
+          version: ${{ steps.semantic_release.outputs.version }}
       - name: Update version in pyproject.toml
         if: steps.semantic_release.outputs.version != ''
-        run: sed -i 's/version = .*/version = "${{ steps.semantic_release.outputs.version }}"/' pyproject.toml
+        run: sed -i '0,/version = .*/s//version = "${{ steps.convert_version.outputs.version }}"/' pyproject.toml
```

Comment:
Only replace the first match, as dependencies can (and do) also have a "version" key specified.